### PR TITLE
UFS/dev PR#216

### PR DESCRIPTION
**Pull request by grantfirl**
Contains the following changes:

- Merges changes from release/public-v7 into main
- adds --no_plots option to tests/cmp_rt2bl.py in order to just do basic RT check

RT differences:
- baseline changes for all suites that use satmedmfvdifq where code changes are exercised (all SCM_GFS_v17_p8_ugwpv1, SCM_GFS_v16_RRTMGP, SCM_GFS_v16, SCM_GFS_v17_p8 tests with convection over ocean)

Note, I performed the RT test on my mac by generating baselines from the top of main and checking against this PR branch using the scripts in test/ci_util.py/cmp_rt2bl.py since I don't think that the baselines on the FTP server are updated.

### CODE REVIEW

**Comment by grantfirl** [comment]
@mkavulich @scrasmussen @dustinswales https://github.com/NCAR/ccpp-physics/pull/1092 has been approved. Could you please review this? Once approved, I'd like to merge this. @mkavulich Then, we need to upload the RT artifact to the FTP server.

File: .gitmodules [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	branch = main
 [submodule "ccpp-physics"]
 	path = ccpp/physics
-	url = https://github.com/NCAR/ccpp-physics
-	branch = main
+	url = https://github.com/grantfirl/ccpp-physics
+	branch = ufs-dev-PR216
```
Comment (scrasmussen) [inline .gitmodules:8]: Just double checking that pointing to your `ufs-dev-PR216` branch is on purpose. (I'm guessing until the CCPP Physics [PR#1092](https://github.com/NCAR/ccpp-physics/pull/1092) is merged?) Everything else looks good to me, thanks!

Comment (grantfirl) [reply .gitmodules:8]: Yes, it's just for testing, just like for UFS. It gets reverted after the ccpp-physics PR is merged.

**Review by dustinswales — APPROVED** [review]

**Comment by mkavulich** [comment]
@grantfirl @dustinswales artifacts have been updated at https://dtcenter.ucar.edu/ccpp/rt/

**Comment by dustinswales** [comment]
Rerunning RTs now via workflow dispatch, https://github.com/NCAR/ccpp-scm/actions/runs/11448914156

**Pull request merged**